### ISSUE TITLE
Button: Button component doesn't pass down aria-haspopup attribute to button element

### ISSUE THREAD

**Issue by bonniegso** [SAP/ui5-webcomponents#4384]
**Describe the bug**
When provided with aria-haspopup="true", the <button> element in the shadowDOM doesn't inherit the attribute from the <Button> component so "has popup" isn't announced by screen readers when button is focused. This attribute is needed to ensure that buttons that trigger popups adhere to a11y standards.

**Isolated Example**
Example: https://4ebcf.csb.app/

**To Reproduce**
1. Go to 'https://4ebcf.csb.app/'
2. Turn on your screenreader
3. Tab to the button
4. Observe that "has popup" is not announced.

**Expected behavior**
When aria-haspopup attribute is provided, it should pass to the <button> element and "has popup" 
should be announced by the screen reader if this attribute is true.

**Screenshots**
<img width="988" alt="Screen Shot 2021-11-23 at 9 01 51 AM" src="https://user-images.githubusercontent.com/68353216/143070076-8af049fe-7c09-4068-82ff-cf1409fbd2bc.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 0.31.21
`@ui5/webcomponents-react` version: 0.131.23
Operating System: MacOS Big Sur
Browser: Google Chrome Version 95.0.4638.69

**Comment by niyap** [comment]
Hello @SAP/ui5-webcomponents-topic-b,

That is a feature request regarding the ui5-button component.
The reporter wants to have a property for adding aria-haspopup attribute. In sap.m.Button, such a property already exists and is called "ariaHasPopup". 
Could you please check and evaluate whether similar property could be added to the ui5-button component?

Kind Regards,
Niya

**Comment by petyabegovska** [comment]
Closed with [feat(ui5-button): implement accessibilityAttributes property](https://github.com/SAP/ui5-webcomponents/pull/4655#)
#4655
